app/api/routers/breeze_buddy/telephony/answer/handlers.py

```
@@ -571,6 +577,42 @@ async def make_response(ws_url: str) -> Response:
+_RECORDING_SETTLE_SECONDS = 0.5
+
+
+async def _start_recording_after_delay(call_id: str, tag: str) -> None:
+    """Wait out Plivo's settle window, then start recording in a worker thread."""
+    try:
+        await asyncio.sleep(_RECORDING_SETTLE_SECONDS)
+        with timed_phase("start_recording_background"):
+            recording_started = await start_call_recording(call_id)
+        if not recording_started:
+            logger.error(f"[{tag}] Recording failed to start for call: {call_id}")
+    except Exception as e:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)